src/torchcodec/decoders/_core/VideoDecoder.h

```diff
@@ -123,7 +123,13 @@ class VideoDecoder {
   // --------------------------------------------------------------------------
   // ADDING STREAMS API
   // --------------------------------------------------------------------------
-
+  enum ColorConversionLibrary {
+    // TODO: Add an AUTO option later.
+    // Use the libswscale library for color conversion.
```

Comment:
Comment looks wrong; should say something about filtergraph.

Reply:
Good catch. Done.